manifests/s/Splunk/ACS/2.5.0/Splunk.ACS.locale.en-US.yaml

```
@@ -0,0 +1,28 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Splunk.ACS
+PackageVersion: 2.5.0
+PackageLocale: en-US
+Publisher: Splunk, Inc.
```

```suggestion
Publisher: Splunk
```
cc @rsatalkar: This must match Control Panel.